source/install/desktop.rst

```diff
@@ -71,3 +69,23 @@ To download the unofficial, community-driven .deb packages for the desktop appli
 2. Open a terminal and execute the command ``sudo dpkg -i mattermost-desktop-<VERSION>-<ARCH>.deb``
 
 3. Open **Dash** (located at top left corner) and input ``mattermost``, then click the Mattermost icon. The Mattermost desktop should open automatically.
+
+Troubleshooting
+--------------------------------------------------
+
+Possible solutions to issues encountered when using the Desktop App.
+
+"Installation has failed" dialog
```

Comment:
I think if you want the labels to show up in the ToC or be automatically included via the `.. contents::` directive, then you have no choice but to use headings. Otherwise, a table would probably be the best way, but tables in reST are really difficult to get right, so a definition list would suffice.